plugins/catalog-backend-module-incremental-ingestion/src/engine/IncrementalIngestionEngine.ts

```diff
@@ -15,18 +15,16 @@
  */
 
 import type { DeferredEntity } from '@backstage/plugin-catalog-backend';
-import {
-  INCREMENTAL_ENTITY_PROVIDER_ANNOTATION,
-  IterationEngine,
-  IterationEngineOptions,
-} from '../types';
+import { IterationEngine, IterationEngineOptions } from '../types';
 import { IncrementalIngestionDatabaseManager } from '../database/IncrementalIngestionDatabaseManager';
 import type { AbortSignal } from 'node-abort-controller';
 import { performance } from 'perf_hooks';
 import { Duration, DurationObjectUnits } from 'luxon';
 import { v4 } from 'uuid';
 import { stringifyError } from '@backstage/errors';
 
+const REMOVAL_THRESHOLD = 5;
```

Comment:
Yep let's make this an external, optional, parameter as mentioned elsewhere

Reply:
Agreed. I'll make it so.

Reply:
Done!